pkg/ingester/shipper.go

```diff
@@ -152,34 +145,45 @@ func (s *Shipper) Sync(ctx context.Context) (uploaded int, err error) {
 			return 0, errors.Wrap(err, "check exists")
 		}
 		if ok {
-			meta.Uploaded = append(meta.Uploaded, m.ULID)
-			uploaded++ // the last upload must have failed, report the block as if it was uploaded successfully now
+			// TODO Whishlist would be great to have a file in the bucket telling us when the block was uploaded.
```

Comment:
That would be last modified time of `meta.json` file (because it's uploaded as last), which can be obtained by calling `bucket.Attributes` instead of `bucket.Exists`. Also we try not to leave "wishlist" TODOs around.

Reply:
Decided to be conservative and simply say it was rencetly uploaded. Removed the wishlist and TIL about bucket.Attributes.